google_api_lib/drive.py

```diff
@@ -0,0 +1,245 @@
+import httplib2
```

Comment:
Is it possible to do `git mv google_api_client.py drive.py` as a separate commit so it's easier to see what the changes are to the file and to preserve the lineage? (This can be part of this PR, just as a separate commit before making the changes to the file. I think if done this way, GitHub should show "google_api_client.py -> drive.py" and show the diff.)